### DISCUSSION TITLE
667FE434-3F45-47A6-829E-F9F5ECACC186

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#140] [General]
# Hoag's Object

This add-on adds Hoag's Object, an unusual ring galaxy in Serpens Caput. It is a ring galaxy, and from Earth it appears nearly face-on. It is very symmetrical for a ring galaxy, unlike other galaxies like Arp 147.

The add-on is made of two parts. There is a ring component of Hoag's Object, which is defined using a CustomTemplate. The core component is defined using a special CMOD model, which is essentially just a bunch of nearly transparent 

https://celestia.mobi/addon?item=667FE434-3F45-47A6-829E-F9F5ECACC186&lang=es

**Comment by Pascualito1008** [comment]
Less than milky way

**Reply by kmjordan122313-lgtm** [reply]
what do you mean less than milky way?

**Comment by kmjordan122313-lgtm** [comment]
this is a cool addon.
